java_tests/src/test/java/org/lsst/sal/LsstDdsQosTest.java

```diff
@@ -16,31 +16,28 @@
  * native libraries and doesn't unload them until the JVM stops.
  */
 @RunWith(PowerMockRunner.class)
-@PrepareForTest(System.class)
+@PrepareForTest(SAL_Test.class)
```

Comment:
At line 15 (a few lines above) you still have a comment:
```
 * This unit test class doesn't make much sense in Java due to the way JNI loads the
 * native libraries and doesn't unload them until the JVM stops.
```
Can you remove that comment now, or does it still apply? I'm no expert on Java and Mockito, but it looks naively, at least, as if these tests are doing useful things.

Reply:
Comment removed. Thanks for catching this.